privacyidea/static/components/config/views/config.events.details.html

```diff
@@ -133,7 +133,7 @@ <h3 translate>Condition</h3>
                         </button>
                     </label>
 
-                    <div class="input-group col-sm-9">
+                    <div class="input-group col-sm-12">
```

Comment:
The columns are divided into 12 sections. So all ``col`` class should add up to 12.

col-sm (column small)...

So in line 123 you have col-sm-3. This should be accompanied by col-sm-9.
So why change this to col-sm-12. This looks wrong to me.

Reply:
![Bildschirmfoto vom 2020-12-11 22-59-05](https://user-images.githubusercontent.com/67051749/101959104-f6052480-3c04-11eb-92c3-1beee5c1f801.png)